packages/devtools-reps/src/object-inspector/actions.js

```diff
@@ -105,6 +86,15 @@ function nodePropertiesLoaded(
   };
 }
 
+function closeObjectInspector() {
+  return async ({ getState, hudProxy }: ThunkArg) => {
```

Comment:
we can pass `hudProxy` into thunk args as well, which should simplify releaseActor as well